src/Init/Notation.lean

```diff
@@ -687,6 +687,27 @@ syntax (name := checkSimp) "#check_simp " term "~>" term : command
 -/
 syntax (name := checkSimpFailure) "#check_simp " term "!~>" : command
 
+/--
+`#discr_tree_key  t` prints the discrimination tree keys for the type of a term `t` or an identifier.
+It uses the default configuration for generating keys.
```

Comment:
I’m still confused as to whether the command gives you the key for the type given, or for the type of the term given. The doc string says the latter (“type of a term `t`”), but the example indicates the former. So probably.

```suggestion
`#discr_tree_key  t` prints the discrimination tree keys for a term `t` (or, if it is a single identifier, the type of that constant).
It uses the default configuration for generating keys.
```